aiohomekit/controller/ip/pairing.py

```diff
@@ -139,6 +139,10 @@ async def _ensure_connected(self):
             raise AccessoryDisconnectedError(
```

Comment:
We currently unconditionally wait for the timeout on failure.

That would be a lot more to refactor so I worked on getting the error message to be better instead.

Reply:
To test this I changed the ip address to start with 224 in the config entry
<img width="711" alt="Screenshot 2023-07-22 at 2 55 47 PM" src="https://github.com/Jc2k/aiohomekit/assets/663432/df46017e-0563-40d3-8d67-3a2447a36b00">


I think this will help a lot with issue reports

Reply:
Previously it would always say it timed out

Reply:
Also pairing failures now show up here as well

Reply:
Although the exception messages are pretty bad